README.md

```diff
@@ -8,7 +8,7 @@
   <a href="https://pypi.org/project/urllib3"><img alt="PyPI Version" src="https://img.shields.io/pypi/v/urllib3.svg?maxAge=86400" /></a>
   <a href="https://pypi.org/project/urllib3"><img alt="Python Versions" src="https://img.shields.io/pypi/pyversions/urllib3.svg?maxAge=86400" /></a>
   <a href="https://discord.gg/urllib3"><img alt="Join our Discord" src="https://img.shields.io/discord/756342717725933608?color=%237289da&label=discord" /></a>
-  <a href="https://codecov.io/gh/urllib3/urllib3"><img alt="Coverage Status" src="https://img.shields.io/codecov/c/github/urllib3/urllib3.svg" /></a>
+  <img alt="Coverage Status" src="./docs/_static/coverage.svg" />
```

Comment:
Since this README is used on PyPI, this link would likely break after being uploaded. Instead let's use:

`https://img.shields.io/badge/coverage-100%25-success`

And we can maintain an `href` and link to our CI as is done below.

Reply:
Sure.

Reply:
The shields.io URL is a great catch, thanks @sethmlarson!